packages/cypress/plugins/cypress-preset.ts

```diff
@@ -7,7 +7,6 @@ import vitePreprocessor from '../src/plugins/preprocessor-vite';
 interface BaseCypressPreset {
   videosFolder: string;
   screenshotsFolder: string;
-  video: boolean;
```

Comment:
would this be breaking if you used to use the `video` property?

Reply:
Talked with Jack and we are standardizing on what cypress is doing which is defaulting video off for v13. people can easily turn it back on in their respective cypress.config.ts.